modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/ErrorAlert/ErrorAlert.js

```diff
@@ -27,59 +27,53 @@ import ErrorAlertExtendedInfo from './ErrorAlertExtendedInfo';
 
 const ErrorAlert = () => {
 	const [showErrorInfo, setShowErrorInfo] = useState(false);
+	const expandInfoHandler = () => {
+		setShowErrorInfo((currentValue) => !currentValue);
+	};
 
 	const {data, error} = useContext(StoreStateContext);
 
-	if (!data || !error) {
-		return null;
-	}
-
-	const {configureGooglePageSpeedURL} = data;
-
-	const {code: statusCode, status} = error;
+	const {configureGooglePageSpeedURL} = data || {};
```

Comment:
I think it's a bit weird to take a prop from an empty object. I guess you want to make it work when data is null, but I would go with another way 😛